src/git/mod.rs

```diff
@@ -121,11 +121,13 @@ pub(crate) fn diff_unstaged(config: &Config, repo: &Repository) -> Res<Diff> {
 pub(crate) fn diff_staged(config: &Config, repo: &Repository) -> Res<Diff> {
     let opts = &mut git2_opts::diff(repo)?;
 
-    let diff = match repo.head() {
+    let mut diff = match repo.head() {
         Ok(head) => repo.diff_tree_to_index(Some(&head.peel_to_tree()?), None, Some(opts))?,
         Err(_) => repo.diff_tree_to_index(None, None, Some(opts))?,
     };
 
+    diff.find_similar(Some(&mut DiffFindOptions::new().renames(true)))?;
```

Comment:
Another available option here is [copies detection](https://docs.rs/git2/latest/git2/struct.DiffFindOptions.html#method.copies). I left it out because I have never seen how that works in practice (in git cli) and I'm only interested in renames for now.

Reply:
Interesting! I've never heard of it myself. Let's leave it :)